### PR TITLE
dosbox-staging: 0.78.1 -> 0.80.1

### PR DESCRIPTION
**Pull request by superherointj**
* dosbox-staging: 0.78.1 -> 0.80.1
* irr1: init 1.9.4

dosbox-staging/0.78.1 is currently broken at master.
This PR is a fix for downstream builds in SDL2.

### CODE REVIEW

File: pkgs/development/libraries/irr1/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, pkgs
```
Comment (wegank) [inline pkgs/development/libraries/irr1/default.nix:4]:
```suggestion
```

Comment (superherointj) [reply pkgs/development/libraries/irr1/default.nix]: Thanks. Fixed.

**Review by wegank — APPROVED** [review]

File: pkgs/applications/emulators/dosbox-staging/default.nix [hunk 2]
```diff
@@ -23,17 +24,19 @@
 , glib
 , libjack2
 , libsndfile
+, irr1
```
Comment (AndersonTorres) [inline pkgs/applications/emulators/dosbox-staging/default.nix:27]: Alphabetic ordering, please.

File: pkgs/development/libraries/irr1/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = {
+    description = "A DSP IIR realtime filter library written in C++";
+    downloadPage = "https://github.com/berndporr/iir1";
+    homepage = "http://berndporr.github.io/iir1/";
+    license = lib.licenses.mit;
+    maintainers = [ ];
```
Comment (AndersonTorres) [inline pkgs/development/libraries/irr1/default.nix:25]: Add myself, just for the case :)

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/irr1/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  meta = {
+    description = "A DSP IIR realtime filter library written in C++";
+    downloadPage = "https://github.com/berndporr/iir1";
+    homepage = "http://berndporr.github.io/iir1/";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.AndersonTorres ];
```
Comment (AndersonTorres) [inline pkgs/development/libraries/irr1/default.nix:25]: platforms?

File: pkgs/applications/emulators/dosbox-staging/default.nix [hunk 5]
```diff
@@ -1,39 +1,42 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, SDL2
-, SDL2_net
-, alsa-lib
+{ alsa-lib
```
Comment (AndersonTorres) [inline pkgs/applications/emulators/dosbox-staging/default.nix:1]: That was not the idea...

lib, stdenv and the fetchers go first; the others go in alphabetical order.

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/development/libraries/irr1/default.nix (hunk 4 above)
Comment (superherointj) [reply pkgs/development/libraries/irr1/default.nix:25]: In theory, all platforms: https://github.com/berndporr/iir1#compilation-from-source
But I haven't tested other platforms than `linux-x86_64`. Can't build them. So `linux-x86_64` then?

**Review by AndersonTorres — APPROVED** [review]
ok

**Comment by superherointj** [comment]
> ok

Thanks.

**Pull request merged**